Feature: Documentation compatibility and improvements (#46)
Atom-Learning/components: src/components/select/Select.mdx

---
title: Select
component: Select
description: Select is a light wrapper around the HTML select element.
---

It adds default styling and the `css` prop.

A `Select` needs to be associated with a label for accessibility purposes, so rather than using the `Select` component directly in a UI, consider using a `SelectField`, which provides a `Label` and displays validation errors. Use this `Select` to compose more complex `Field` type components.

```jsx preview center
<Select
  options={[
    { value: 1, label: 'Apples' },
    { value: 2, label: 'Bananas' }
  ]}
  css={{ width: 300 }}
/>
```

## Accessibility

> For accessibility purposes the component needs to have the `id` prop set, to link it to the label it is associated with. If those are not available, please add an `aria-label` to ensure that the component remains accessible

Reviewer: ```suggestion
For accessibility purposes the component needs to have the `id` prop set, to link it to the label it is associated with. If a label is not available, please add an `aria-label` to ensure that the component remains accessible
```
